rllib/algorithms/iql/torch/default_iql_torch_rl_module.py

```diff
@@ -0,0 +1,80 @@
+import gymnasium as gym
+from typing import Any, Dict, Optional
+
+from ray.rllib.algorithms.iql.default_iql_rl_module import DefaultIQLRLModule
+from ray.rllib.algorithms.iql.iql_learner import VF_PREDS_NEXT, QF_TARGET_PREDS
+from ray.rllib.algorithms.sac.torch.default_sac_torch_rl_module import (
+    DefaultSACTorchRLModule,
+)
+from ray.rllib.core.columns import Columns
+from ray.rllib.core.models.base import ENCODER_OUT
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.framework import try_import_torch
+from ray.rllib.utils.typing import TensorType
+
+torch, nn = try_import_torch()
+
+
+class DefaultIQLTorchRLModule(DefaultSACTorchRLModule, DefaultIQLRLModule):
+
+    framework: str = "torch"
+
+    @override(DefaultSACTorchRLModule)
+    def _forward_train(self, batch: Dict, **kwargs) -> Dict[str, Any]:
+
+        if self.inference_only:
```

Comment:
dumb question. Should we move this logic into all RLModules' `_forward_train()` by putting this if-block into the super class?

Reply:
Great catch @sven1977 ! This is actually already implemented in `RLModule.forward_train`, we just have to call `super`